TrackMyStuffApp/ClientApp/src/app/app.module.ts

```diff
@@ -32,7 +33,13 @@ import { LoadingComponent } from './shared/components/loading/loading.component'
             { path: 'item', component: ItemComponent }
         ], { relativeLinkResolution: 'legacy' }),
         BrowserAnimationsModule,
-        AngularMaterialModule
+        AngularMaterialModule,
+        ToastrModule.forRoot({
```

Comment:
These are *global* options for setting the toaster behavior.

Please read more on [options here.](https://www.npmjs.com/package/ngx-toastr)